eventlet/support/afinet.py

```diff
@@ -0,0 +1,46 @@
+"""
+Test availabilty of AF_INET6 on a machine
+
+Uses socket API to determine configuration
+"""
+
+from socket import *
+
+def check_ipv6_lo_addr():
```

Comment:
What's the purpose of this check anyway? The big picture, I mean. Suppose my machine is able to listen `::1`, why would you hack Eventlet to prefer IPv6 while rest of system doesn't?